src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizations/CodeBlockNodeViewCustomization.cs

```diff
@@ -9,9 +9,11 @@ namespace Dynamo.Wpf
 {
     public class CodeBlockNodeViewCustomization : INodeViewCustomization<CodeBlockNodeModel>
     {
+        private CodeBlockEditor cbe;
```

Comment:
Why make this a member field when it is used as a local var in the method?

Reply:
I think I originally called Dispose on it but reverted the change, will revert